web/scm/src/app/side-panel/side-panel.component.html

```diff
@@ -1 +1,86 @@
-<p>side-panel works!</p>
+<form [formGroup]="form" (submit)="onFormSubmit()">
```

Comment:
What if we can have more than 1 form on the page? Can we name it like filterForm? WDYT?